docs/reference/contrib/modeladmin/base_url.rst

```diff
@@ -0,0 +1,22 @@
+======================================
+Customising the base URL path
+======================================
+
+You can use the following attributes and methods on the ``ModelAdmin`` class to
+alter the base URL path used to represent your model in Wagtail's admin area.
+
+.. contents::
+    :local:
+    :depth: 1
+
+.. _modeladmin_base_url_path:
+
+-----------------------------
+``ModelAdmin.base_url_path``
+-----------------------------
+
+**Expected value**: A string.
+
+Set this attribute to a string value to override the default base URL path
+used for the model to ``admin\{base_url_path}``. If not set, the base URL path
+will be ``admin\{app_label}\{model_name}``

```

Comment:
Please can all new documentation be written in markdown. 
https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html